data/json/recipes/recipe_ammo.json

```diff
@@ -4715,5 +4736,36 @@
       [ "chem_sulphur", 10 ]
     ]
   ]
+},{
+  "type" : "recipe",
+  "result": "gas_chloramine",
```

Comment:
Wait, this is a recipe that produces gas and then allows the survivor to pour it into a bottle?